dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -131,8 +131,11 @@ void crash_signal_intercepted(int signal, const char *signal_name, const char *e
 
     // dump the log into a new file
     char filename[200];
+    #pragma clang diagnostic push
+    #pragma clang diagnostic ignored "-Wformat"
```

Comment:
You should probably add a comment explaining why you choose to ignore the warning rather than fix it.